bool enabled
bool lamp
int16 servo_x
int16 servo_y
int8 line_start_x
int8 line_start_y
int8 line_end_x
int8 line_end_y
int8 line_index
int8 line_flags
int8 num_intersections
int8 intersection_x
int8 intersection_y
int8 intersection_index
int16 intersection_angle
